2018-10-25 02:30:22,171 [1] INFO  SkieerWeb - RefundJob starting.
2018-10-25 02:30:22,203 [13] INFO  SkieerWeb - RefundJob checking.
2018-10-25 02:30:22,203 [13] INFO  SkieerWeb - IsEnabledRefund = False
2018-10-25 02:30:22,203 [13] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-10-25 02:30:31,828 [6] INFO  SkieerWeb - [Braintree Webhook Received 2018/10/24 10:14:56] | Kind: subscription_charged_successfully
2018-10-25 02:30:32,093 [6] INFO  SkieerWeb - [WebhookNotification]Subscription not found.SubscriptionId:dc4a89eb-6725-4196-8948-948e2ceeb802,Kind:subscription_charged_successfully
2018-10-25 03:30:08,593 [1] INFO  SkieerWeb - RefundJob starting.
2018-10-25 03:30:08,609 [11] INFO  SkieerWeb - RefundJob checking.
2018-10-25 03:30:08,609 [11] INFO  SkieerWeb - IsEnabledRefund = False
2018-10-25 03:30:08,609 [11] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-10-25 03:30:10,625 [10] INFO  SkieerWeb - [Braintree Webhook Received 2018/10/24 10:14:56] | Kind: subscription_charged_successfully
2018-10-25 03:30:10,687 [10] INFO  SkieerWeb - [WebhookNotification]Subscription not found.SubscriptionId:dc4a89eb-6725-4196-8948-948e2ceeb802,Kind:subscription_charged_successfully
2018-10-25 11:00:22,531 [1] INFO  SkieerWeb - RefundJob starting.
2018-10-25 11:00:22,546 [12] INFO  SkieerWeb - RefundJob checking.
2018-10-25 11:00:22,546 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-10-25 11:00:22,546 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-10-25 11:00:29,281 [10] INFO  SkieerWeb - [Braintree Webhook Received 2018/10/25 10:59:23] | Kind: subscription_charged_successfully
2018-10-25 11:00:30,343 [10] ERROR SkieerWeb - GetSubscriptionById Error.SubscriptionId:6794a131-a73d-4094-a263-4c38ea995bd0
System.ServiceModel.EndpointNotFoundException: 没有终结点在侦听可以接受消息的 http://192.168.0.103:8017/ChargeService。这通常是由于不正确的地址或者 SOAP 操作导致的。如果存在此情况，请参见 InnerException 以了解详细信息。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 192.168.0.103:8017
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   --- 内部异常堆栈跟踪的结尾 ---

Server stack trace: 
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   在 System.ServiceModel.Channels.HttpOutput.Send(TimeSpan timeout)
   在 System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.SendRequest(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Octoparse.Web.SkieerBillingService.ICharging.GetSubscriptionById(String subscriptionId)
   在 Octoparse.Web.Controllers.SubscribeController.BtHandler()
2018-10-25 11:00:30,359 [10] INFO  SkieerWeb - [WebhookNotification]Subscription not found.SubscriptionId:6794a131-a73d-4094-a263-4c38ea995bd0,Kind:subscription_charged_successfully
2018-10-25 13:12:16,781 [1] INFO  SkieerWeb - RefundJob starting.
2018-10-25 13:12:16,781 [11] INFO  SkieerWeb - RefundJob checking.
2018-10-25 13:12:16,796 [11] INFO  SkieerWeb - IsEnabledRefund = False
2018-10-25 13:12:16,796 [11] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-10-25 13:47:28,546 [1] INFO  SkieerWeb - RefundJob starting.
2018-10-25 13:47:28,562 [12] INFO  SkieerWeb - RefundJob checking.
2018-10-25 13:47:28,562 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-10-25 13:47:28,562 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-10-25 14:18:53,343 [1] INFO  SkieerWeb - RefundJob starting.
2018-10-25 14:18:53,343 [12] INFO  SkieerWeb - RefundJob checking.
2018-10-25 14:18:53,343 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-10-25 14:18:53,343 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-10-25 14:18:58,921 [10] ERROR SkieerWeb - user login error
System.ServiceModel.EndpointNotFoundException: There was no endpoint listening at http://192.168.0.103:8016/SkieerMembershipService that could accept the message. This is often caused by an incorrect address or SOAP action. See InnerException, if present, for more details. ---> System.Net.WebException: Unable to connect to the remote server ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 192.168.0.103:8016
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- End of inner exception stack trace ---
   at System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   at System.Net.HttpWebRequest.GetRequestStream()
   at System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   --- End of inner exception stack trace ---

Server stack trace: 
   at System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   at System.ServiceModel.Channels.HttpOutput.Send(TimeSpan timeout)
   at System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.SendRequest(Message message, TimeSpan timeout)
   at System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   at System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   at System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   at System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   at Octoparse.Web.SkieerMembershipService.ISkieerMembershipService.ValidateUser(String username, String password)
   at Octoparse.Web.Controllers.AccountController.SignIn(LoginModel model, String returnUrl)
2018-10-25 22:39:17,406 [1] INFO  SkieerWeb - RefundJob starting.
2018-10-25 22:39:17,421 [11] INFO  SkieerWeb - RefundJob checking.
2018-10-25 22:39:17,421 [11] INFO  SkieerWeb - IsEnabledRefund = False
2018-10-25 22:39:17,421 [11] INFO  SkieerWeb - RefundJob waiting 30 minutes.
